README.md

```diff
@@ -18,9 +18,9 @@ The service account json in plain text, provided via a secret, etc.
 
 **Required:** The package name, or Application Id, of the app you are uploading
 
-### `releaseFile`
+### `releaseFiles`
```

Comment:
Please update the `actions.yml` file to reflect this input change

Reply:
Have I not updated it correctly? I've renamed releaseFile to releaseFiles and updated the description in the action config already.